### PR TITLE
save vm endpoints in db

### PR DESCRIPTION
**Pull request by imakunin**

### CODE REVIEW

File: lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java [hunk 1]
```diff
@@ -360,7 +319,7 @@ public Result deallocate(Vm vm) {
 
             var statusDetails = client.pods()
                 .inNamespace(ns)
-                .withLabelSelector(KuberLabels.LZY_VM_ID_LABEL + "=" + vmId.toLowerCase(Locale.ROOT))
+                .withName(podName)
```
Comment (imakunin) [inline lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java:322]: @Hoxo , I did it :)

File: lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorApiTestBase.java [hunk 2]
```diff
@@ -121,8 +106,8 @@ protected void setUp() throws IOException {
 
         kubernetesServer = new KubernetesMockServer(new MockWebServer(), new ConcurrentHashMap<>(), false);
         kubernetesServer.init(InetAddress.getLoopbackAddress(), 0);
-        kubernetesServer.expect().post().withPath("/api/v1/pods")
-            .andReturn(HttpURLConnection.HTTP_OK, new PodListBuilder().build()).always();
+//        kubernetesServer.expect().post().withPath("/api/v1/pods")
```
Comment (Hoxo) [inline lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorApiTestBase.java:109]: I think it's not necessary anymore

Comment (imakunin) [reply lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorApiTestBase.java]: fixed

File: lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java [hunk 3]
```diff
@@ -397,68 +356,76 @@ public Result deallocate(Vm vm) {
...
-                meta.put(NODE_NAME_KEY, nodeName);
-                if (instanceId != null) {
-                    meta.put(NODE_INSTANCE_ID_KEY, instanceId);
-                }
-            } else {
+            if (pod == null) {
                 throw new RuntimeException("Cannot get pod with name " + podName + " to get addresses");
             }
+
+            final var nodeName = pod.getSpec().getNodeName();
```
Comment (Hoxo) [inline lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java:373]: Not sure if it's okay to expect that nodeName will be not null. For example, pod can be in scheduling state, waiting for free nodes.

Comment (imakunin) [reply lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java:373]: this method (updateAllocatedVm) is called _after_ allocation only, on callback from allocator agent

Comment (Hoxo) [reply lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberVmAllocator.java:373]: Oh, now I see. But it's would be better, if exception is different from simple NPE

**Pull request merged**